src/Arcus.Templates.WebApi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/aspnet:3.1.20-alpine3.14 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0.0-alpine3.14 AS base
```

Comment:
I think we should only do this when we migrate to .NET 6
In other words, I would'nt do this right now